integration-tests/src/test_loop/utils.rs

```diff
@@ -0,0 +1,77 @@
+use crate::test_loop::env::TestData;
+use itertools::Itertools;
+use near_async::messaging::SendAsync;
+use near_async::time::Duration;
+use near_client::test_utils::test_loop::ClientQueries;
+use near_network::client::ProcessTxRequest;
+use near_primitives::test_utils::create_user_test_signer;
+use near_primitives::transaction::SignedTransaction;
+use near_primitives::types::AccountId;
+use std::collections::HashMap;
+
+pub(crate) const ONE_NEAR: u128 = 1_000_000_000_000_000_000_000_000;
+
+/// Execute money transfers within given `TestLoop` between given accounts.
```

Comment:
also add comment saying this also runs the chain long enough for the money transfers to be completed/finalized.